app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command: str) -> None:
+    file_name = command.split()
+    if len(file_name) == 3:
```

Comment:
Write one if condition and avoid nesting in your code when possible. You could use `all([...])` and place all your conditions inside of the list to organize your code better.